### ISSUE TITLE
support 1.1

### ISSUE THREAD

**Issue by yannick** [D-Programming-Deimos/openssl#34]
a few methods are not supported. e.g. `OPENSSL_init_ssl`

**Comment by jeking3** [comment]
Same for openssl-1.0.2...

**Comment by rinwi** [comment]
I have made a PR with some digest functions (#38).

I can make another PR after that one if merged with others if people are interested.

**Comment by jeking3** [comment]
Also `TLS_method()`, `TLS_server_method()`, `TLS_client_method()` were added, and `SSLv23_*_method()` were removed (previously deprecated, but I'm getting undefined symbol errors with openssl-1.1.0g on Ubuntu Bionic 18.04 LTS.

**Comment by jeking3** [comment]
@WalterBright although the PR made things compatible, until there is a tagged release one can use as a dependency perhaps this should remain open?

**Comment by CyberShadow** [comment]
Is anything missing before we can create the tag?
